CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.4)
+
+project(acc)
+
+set(SOURCE_FILES
```

Comment:
This is kind of an anti-pattern in modern CMake (where generally speaking variables can be avoided).  I would suggest either putting this list in add_executable directly or bumping the required CMake version to 3.1 in order to use target_sources.